cli/smartcontract/testdata/rpcbindings/notifications/rpcbindings_guessed.out

```diff
@@ -80,7 +80,6 @@ type LedgerWitnessRule struct {
 // Unnamed is a contract-specific unnamed type used by its methods.
 type Unnamed struct {
 	I *big.Int
-	B bool
```

Comment:
But why?

Reply:
It's the consequence of https://github.com/nspcc-dev/neo-go/issues/3088, and it was added to the commit message accidently. Removed.